docs/semgrep-app/integrations.md

```diff
@@ -8,12 +8,97 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Integrations
 
+Semgrep App contains third party integrations to allow users to add data from Semgrep to other tools that are part of their workflows.
+
+Currently, Semgrep App integrates with the following tools:
+
+| Tool | Tier availability |
+| ---- | ---------------- |
+| Slack | Community (Free) |
+| Email | Community (Free) |
+| Jira | Team |
+| Amazon S3 | Team |
+| Webhook | Team |
+
+## Where are integrations found?
+
+After logging into your Semgrep App account, you can find **Integrations** as a tab within **Settings.**
+
+![Screenshot of Semgrep's "Create New Integration Channel" menu](../img/integration-firstview.png)<br />
+
+View your saved integration channels by clicking on the **gear** icon within the **Rule board**. Click on the toggles to turn notifications on or off for each channel.
+
+![Screenshot of the Semgrep rule board with integrations](../img/integration-ruleboard.png)<br />
+
+## Slack
+
+Integrating with Slack allows Semgrep to post messages after each scan about new findings to a channel on your Slack workspace.
+
+To set it up, you'll need to create an app within Slack and enable that app's **Incoming Webhooks** feature. By creating a webhook, you'll have a URL which serves as Semgrep's endpoint to your Slack workplace. The guide below will walk you through those steps.
+
+1. Make sure that you're logged into your Slack account. You can enter your login details [here.](https://slack.com/workspace-signin)
+2. Follow this [link](https://api.slack.com/apps?new_app=1) to create an app in your Slack workplace. Select **From scratch** and fill in the fields for your app's name and workspace.
+
+![Screenshot of a Slack form to fill out new app details](../img/slack-create-app-name.png)<br />
+
+3. You will be redirected to your new app's settings page. Alternatively, you can find this page by viewing your apps [here](https://api.slack.com/apps) and selecting your integration app from the menu.
+4. Click on **Incoming Webhooks** on the left menu, then enable the toggle.
+5. Once you've enabled **Incoming Webhooks**, you'll either be able to generate a **Webhook URL** or request one from a workspace owner.
```

Comment:
```suggestion
6. When you enabled **Incoming Webhooks**, follow one of these steps:
    - Generate a **Webhook URL**.
    - Request such a URL from a workspace owner.
```

Reply:
I agree that "once" should not be used as a synonym for "after" and that I should not have used a contraction in this case. I'd prefer "After" over "When" though, so editing that now then committing :)